core/src/main/res/layout/activity_kiwix_error.xml

```diff
@@ -25,12 +25,13 @@
 
   <Button
     android:id="@+id/reportButton"
-    android:layout_width="120dp"
+    android:layout_width="wrap_content"
     android:layout_height="wrap_content"
     android:layout_marginStart="8dp"
     android:layout_marginEnd="8dp"
     android:layout_marginBottom="32dp"
-    android:background="@color/alabaster_white"
```

Comment:
Why did you remove the background color?

Reply:
The background in this button is not required as it was creating additional padding which made the height of this button not equal with the others